docs/source/en/tasks/image_text_to_text.md

```diff
@@ -0,0 +1,208 @@
+<!--Copyright 2024 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# Image-text-to-text
+
+[[open-in-colab]]
+
+Image-text-to-text models, also known as vision language models, are language models that can take image input. These models can tackle various tasks, from visual question answering to image segmentation. This task has many similarities to `image-to-text` with some overlapping use cases like image captioning. `image-to-text` models take only image input and often accomplish a specific task, meanwhile `image-text-to-text` models take open-ended text and image inputs and are more generalist models.
+
+In this guide, we will do a brief overview of these models and show how to use transformers to infer `image-text-to-text` models. To begin with, there are multiple types of vision language models: base models that are used for fine-tuning, chat fine-tuned (or chatty) models for conversation and instruction fine-tuned models. Here we will see how to infer an instruction-tuned model. 
+
+Let's begin installing the dependencies.
+
+```bash
+pip install -q transformers accelerate flash_attn 
+```
+
+Let's initialize the model and the processor. 
+
+```python
+from transformers import AutoProcessor, Idefics2ForConditionalGeneration
+import torch
+
+device = torch.device("cuda")
+model = Idefics2ForConditionalGeneration.from_pretrained(
+        "HuggingFaceM4/idefics2-8b",
+        torch_dtype=torch.bfloat16,
+        _attn_implementation="flash_attention_2",
+    ).to(device)
+
+processor = AutoProcessor.from_pretrained("HuggingFaceM4/idefics2-8b")
+```
+
+This model has a chat template format that's required to input. Moreover, the model can also input multiple images in single conversation or a single message. We will now prepare the inputs. 
```

Comment:
May be helpful to link to the [Templates for chat models](https://moon-ci-docs.huggingface.co/docs/transformers/pr_31777/en/chat_templating) doc to help users learn more.

```suggestion
This model has a chat template format that's required for the input. Moreover, the model can also accept multiple images as input in a single conversation or message. We will now prepare the inputs. 
```